.github/workflows/tests-php.yml

```diff
@@ -0,0 +1,82 @@
+name: 'Codeception Tests'
+on:
+  pull_request:
+    paths:
+      - 'common/src/**.php'
+      - 'common/*.php'
+      - 'src/**.php'
+      - 'tests/**'
+      - '*.php'
+      - 'composer.json'
+      - 'codeception.*.yml'
+      - '.github/workflows/tests-php.yml'
```

Comment:
same as above, curious why we wouldn't run these anytime we have a PR.

Reply:
The idea here is that if we aren't touching anything the PHP or the PHP tests rely on, it's not worth running/waiting on the action itself.

Reply:
If we change any code in common, we don't change any PHP code, but can affect tests.

Reply:
I'm curious about which files could cause that, I'm open to just making it open for all changes but I felt like this would be a conservative use of actions so resources weren't wasted.